Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -8536,16 +8557,15 @@ - (BOOL)deferringGestureRecognizer:(WKDeferringGestureRecognizer *)deferringGest
         if (gestureRecognizer == [_textInteractionAssistant loupeGesture])
             return YES;
 
-        if ([gesture isKindOfClass:UITapGestureRecognizer.class]) {
-            UITapGestureRecognizer *tapGesture = (UITapGestureRecognizer *)gesture;
+        if ([gestureRecognizer isKindOfClass:UITapGestureRecognizer.class]) {
+            UITapGestureRecognizer *tapGesture = (UITapGestureRecognizer *)gestureRecognizer;
```

Comment:
Might be nice to use `dynamic_objc_cast`.

Reply:
Will do!